.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dockerFile": "Dockerfile",
+  "extensions": [
+    "pgourlain.erlang"
```

Comment:
Does this still makes sense? That extension uses a different Language Server, so this may cause weird interactions. Or?

Reply:
I've removed it, now you can use the `erlang_ls` extension to work on `erlang_ls` ;)